### DISCUSSION TITLE
Memory leak?

### DISCUSSION THREAD

**Discussion by MrWyss** [louislam/dockge#59] [Ask for Help]
I noticed my pi4 becomes a bit unresponsive after a while running dockge, so that I have to hard reset it. 
is there a possible memory leak? 

I do see this node process, i pretty sure it's from the dockge container. 

```
PID    PPID USER     %MEM %CPU CMD
98438   98422 root      1.5  2.2 /usr/local/bin/node --require /pnpm/global/5/.pnpm/tsx@3.14.0/node_modules/tsx/dist/preflight.cjs --import file:///pnpm/global/5/.pnpm/tsx@3.14.0/node_modules/tsx/dist/loader.mjs ./backend/index.ts
```

```
  PID USER      PR  NI    VIRT    RES    SHR S  %CPU  %MEM     TIME+ COMMAND                                             
98438 root      20   0   25.4g 123388  50148 S   2.0   1.5   0:11.42 node
```
_btw: other than that, I love it._

**Comment by louislam** [comment]
Currently, some log terminals are not closed after you leave the page.

I am trying to improve this: https://github.com/louislam/dockge/pull/60
